.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
don't commit your gitignore. please manage it on your side.
https://gist.github.com/subfuzion/db7f57fff2fb6998a16c